libs/@guardian/core-web-vitals/package.json

```diff
@@ -8,13 +8,13 @@
 		"@guardian/libs": "12.0.0",
 		"tslib": "2.4.1",
 		"typescript": "4.3.2",
-		"web-vitals": "2.0.0"
+		"web-vitals": "3.1.1"
 	},
 	"peerDependencies": {
 		"@guardian/libs": "^12.0.0",
 		"tslib": "^2.4.1",
 		"typescript": "^4.3.2",
-		"web-vitals": "^2.0.0"
+		"web-vitals": "^2.0.0 || ^3.0.0"
```

Comment:
the dev dep version doesn't align with the min peerdep range. if you want to bump to 3, it might be better to specify `^3.0.0` peerdep?

Reply:
Yeah I thought that but foolishly deferred to Renovate, updated.